certified-connectors/Luware Nimbus/apiProperties.json

```diff
@@ -0,0 +1,94 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "<< Please add your client id >>",
+          "scopes": [ "User.Read" ],
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "<< Please add your client id >>"
+            }
+          },
+          "uiDefinition": {
```

Comment:
Could you please fix the apiProperties.json issue by referring the schema from this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/schemas/paconn-apiProperties.schema.json

Reply:
Hi @vmanoharas 

I have two questions about this. 

1. @{level=Error; message=Value "GlobalPerConnector" is not defined in enum.; valuePath=line: 10 position: 46 path: '[0].properties.connectionParameters.token.oAuthSettings.redirectMode'; value=GlobalPerConnector}
We changed this from "Global" to "GlobalPerConnector", but according to the schema, only "Global" and "Direct" are allowed. The change was made based on your comment on Friday: https://github.com/microsoft/PowerPlatformConnectors/pull/3097#discussion_r1397251539 

2. @{level=Critical; message=Invalid value for the description property within the UI Definition object.; valuePath=$.properties.connectionParameters.token:TenantId.uiDefinition.description; value=The tenant ID of for the Azure Active Directory application}
According to the schema,  "The tenant ID of for the Azure Active Directory application" is the only allowed value here.

Can you please advice on what are the correct values in these two cases, so the file doesn't fail the validation?

Reply:
Hello @bleul, 

Please accept my apology, if my comment is confusing.  For #1 please GlobalPerConnector error, this need to be updated from our end. 

For # 2 - could you please remove description and displayname?

<img width="474" alt="image" src="https://github.com/microsoft/PowerPlatformConnectors/assets/137378994/d5e01432-ee52-4eed-b20a-5472b6f65879">

Reply:
Hi @vmanoharas 

apiProperties.json was updated accordingly.

Reply:
Thank you very much @bleul